chain/src/tests/basic.rs

```diff
@@ -216,7 +216,7 @@ fn test_transaction_conflict_in_same_block() {
     let mut chain = MockChain::new(parent.clone(), shared.consensus());
     chain.gen_empty_block(&mock_store);
 
-    let last_cell_base = &chain.tip().transactions()[0];
+    let last_cell_base = &shared.consensus().genesis_block().transactions()[1];;
```

Comment:
why change the tip to genesis?

Reply:
1~11 block's cellbase has empty output,  can't be used to generate a new transaction. this broken many test.
I create some cell in genesis on mockchain to replace it.

Reply:
ok